docs/100-upgrade/migration-guides/3.4-3.5.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 3.4 -> 3.5
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.4
+  to 3.5
+---
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/*@3.5.0"
+```
+
+## Configuration updates
+
+In this new release, we've introduced some changes on how some configurations
+are set.
+
+### Cache
+
+We've introduce new cache related to the `config` object in `FrontCommerceApp`.
+These are now defined with a config provider, it allow you to change the
+behaviour of the configuration computing.
+
+| Config path                     | Env Variable                                |
+| ------------------------------- | ------------------------------------------- |
+| `config.cache.apiToken`         | `FRONT_COMMERCE_CACHE_API_TOKEN`            |
+| `config.cache.ignoreCacheRegex` | `FRONT_COMMERCE_BACKEND_IGNORE_CACHE_REGEX` |
+
+### Magento 1/2
+
+We have updated the way we use the Magento 1/2 configuration. It's now possible
+to access Magento OAuth configuration with:
+
+| Config path                              | Env Variable                                                                                   |
+| ---------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| `config.magento.oauth.accesToken`        | `FRONT_COMMERCE_MAGENTO1_ACCESS_TOKEN` OR `FRONT_COMMERCE_MAGENTO2_ACCESS_TOKEN`               |
+| `config.magento.oauth.accessTokenSecret` | `FRONT_COMMERCE_MAGENTO1_ACCESS_TOKEN_SECRET` OR `FRONT_COMMERCE_MAGENTO2_ACCESS_TOKEN_SECRET` |
+| `config.magento.oauth.consumerKey`       | `FRONT_COMMERCE_MAGENTO1_CONSUMER_KEY` OR `FRONT_COMMERCE_MAGENTO2_CONSUMER_KEY`               |
+| `config.magento.oauth.consumerSecret`    | `FRONT_COMMERCE_MAGENTO1_CONSUMER_SECRET` OR `FRONT_COMMERCE_MAGENTO2_CONSUMER_SECRET`         |
+
+### Image Resizer
+
+Display of dev error message is now handled with a configProvider and the
+configuration can be accessible like this:
```

Comment:
Remove the "accessible like this" part, because this is not how they would access it, they would access it via the `FrontCommerceApp.config.XXX`

```suggestion
Display of dev error message is now handled with a configProvider:
```